src/Sarif/Writers/ConsoleLogger.cs

```diff
@@ -125,7 +122,7 @@ public void Log(ReportingDescriptor rule, Result result, int? extensionIndex = n
             WriteLineToConsole(GetMessageText(_toolName, physicalLocation?.ArtifactLocation?.Uri, physicalLocation?.Region, result.RuleId, message, result.Kind, result.Level));
         }
 
-        private static string GetMessageText(
+        public static string GetMessageText(
```

Comment:
>public [](http://example.com/codeflow?start=8&length=6)

I've elevated this helper to public to unblock another change. My sense though is there would be a better api, probably driven by a single result instance that would be better as a public offering.

And so I'm not documenting this change in the release notes. :) I will go back to it.